.env.example

```
@@ -46,8 +46,8 @@ SIBYL_FALKORDB_BROWSER_PORT=3335
 # Leave SIBYL_SURREAL_URL empty and set SIBYL_SURREAL_DATA_DIR to use embedded mode.
 SIBYL_SURREAL_URL=ws://localhost:8000/rpc
 # SIBYL_SURREAL_DATA_DIR=./.moon/cache/surreal-dev
-SIBYL_SURREAL_USERNAME=root
-SIBYL_SURREAL_PASSWORD=root
+SIBYL_SURREAL_USERNAME=change-me
+SIBYL_SURREAL_PASSWORD=change-me-strong-password
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SIBYL_SURREAL_PASSWORD key should go before the SIBYL_SURREAL_URL key

(UnorderedKey)